frontend/index.jsx

```diff
@@ -35,6 +35,7 @@ class App extends React.Component {
       aesKey: null,
       tab: 'my'
     }
+    this.readSubscribersImages = this.readSubscribersImages.bind(this);
```

Comment:
This seems unnecessary? You might want to do that at the exact place you need that ;)

Reply:
In its defense... I think this pattern is used in a lot of React projects. It kind of puts the boilerplate in one place, and also it seems unlikely to me that we'd need to use this method without the binding.

Reply:
this looks quite weird for me too I mean, "this.readSubscribersImages" should have bound "this" within its constructor closure anyway. If the newly bound "this" is sth different that the original "this" this method should be rewritten :D